src/eva/core/data/splitting/stratified.py

```diff
@@ -0,0 +1,52 @@
+"""Functions for stratified splitting."""
+
+from typing import Any, List, Sequence, Tuple
+
+import numpy as np
+
+
+def stratified_split(
+    samples: Sequence[Any],
+    targets: Sequence[Any],
+    train_ratio: float,
+    val_ratio: float,
+    test_ratio: float | None = None,
```

Comment:
maybe just default test_ratio to 0, then we don't need `test_ratio is not None and test_ratio > 0.0` later

Reply:
good point. I updated this now accordingly